app/controllers/application_controller.rb

```diff
@@ -1890,13 +1890,10 @@ def controller_for_common_methods
       "generic_object"
     when "ansible_playbook", "workflow"
       "embedded_configuration_script_payload"
-    when "workflow_repository"
-      "ansible_repository"
-    # TODO: Update manageiq/db/fixtures/miq_product_features.yml by replacing ansible_credential_tag with embedded_automation_manager_credential_tag
-    # when "workflow_credential", "ansible_credential"
-    #  "embedded_automation_manager_credential"
-    when "workflow_credential"
-      "ansible_credential"
+    when "workflow_repository", "ansible_repository"
```

Comment:
I added `ansible_repository` to match the pattern we have in the next `when`:

```ruby
when "workflow_credential", "ansible_credential"
```

It didn't exist here before.  Feel free to drop it.